app/domain/synchronize/mailchimp/synchronizator.rb

```diff
@@ -27,7 +27,11 @@ def initialize(mailing_list, with_default_tag: true)
         @default_tag = format(DEFAULT_TAG, @list.id) if with_default_tag
       end
 
-      def perform
+      def perform # rubocop:disable Metrics/AbcSize
+        unless client.ping_api(error_handler: lambda { |e| result.exception = e })
```

Comment:
Ich fände besser, wenn das noch in eine Methode  extrahiert werden würde, sowas wie 

`return result unless ping_successful?`